cmd/porter/bundle.go

```diff
@@ -80,7 +80,7 @@ The docker driver builds the bundle image using the local Docker host. To use a
 	f.StringVar(&opts.Name, "name", "", "Override the bundle name")
 	f.StringVar(&opts.Version, "version", "", "Override the bundle version")
 	f.StringVarP(&opts.File, "file", "f", "",
-		"Path to the Porter manifest. Defaults to `porter.yaml` in the current directory.")
+		"Path to the Porter manifest. When build context directory is specified, relative path will be resolved using the build context directory as the root directory. Defaults to `porter.yaml` in the current directory.")
```

Comment:
I think we can simplify this a bit since the build context directory defaults to the working directory.

```suggestion
		"Path to the Porter manifest. The path is relative to the build context directory. Defaults to porter.yaml in the current directory.")
```

Reply:
I wanted to make sure that users understand that if a absolute path is passed in, then it won't be relative to the build context directory. How about we say
`When a relative path is provided, the path is relative to the build context directory"

Reply:
The build context directory (--dir) defaults to the current directory. So when you say "relatative to the build context directory" I think that covers both cases, no? 

I just double checked and --dir doesn't say "Defaults to the current directory". How about we add that (it should have been included so that's an oversight)? I'm trying to avoid complex explanations, and explaining the --dir flag on the helptext for the --file flag.